schemas/biomarkers.json

```diff
@@ -0,0 +1,481 @@
+{
+  "name": "biomarkers",
+  "description": "The collection of data elements describing a donor's biomarker tests. A biomarker is a biological molecule found in blood, other body fluids, or tissues that is indicative of the presence of cancer in the body. Each row should include biomarker tests associated with a particular clinical event or time interval.",
+  "meta": {
+    "parent": "donor"
+  },
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.program_id",
+        "displayName": "Program ID"
+      },
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.submitter_donor_id",
+        "displayName": "Submitter Donor ID"
+      },
+      "restrictions": {
+        "required": true,
+        "regex": "#/regex/submitter_id"
+      }
+    },
+    {
+      "name": "submitter_specimen_id",
+      "description": "Unique identifier of the specimen, assigned by the data provider.",
+      "valueType": "string",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.submitter_specimen_id",
+        "displayName": "Submitter Specimen ID"
+      },
+      "restrictions": {
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "name": "submitter_primary_diagnosis_id",
+      "valueType": "string",
+      "description": "If the biomarker test was done at the time of primary diagnosis, then indicate the associated submitter_primary_diagnosis_id here.",
+      "meta": {
+        "displayName": "Submitter Primary Diagnosis ID",
+        "primaryId": true
+      },
+      "restrictions": {
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "name": "submitter_treatment_id",
+      "valueType": "string",
+      "description": "If the biomarker test was done at the initiation of a specific treatment regimen, indicate the associated submitter_treatment_id here.",
+      "meta": {
+        "primaryId": true,
+        "displayName": "Submitter Treatment ID"
+      },
+      "restrictions": {
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "description": "If the biomarker test was done during a follow up event, then indicate the associated submitter_follow_up_id here.",
```

Comment:
`follow-up` to be consistent with the current dictionary terminology

Reply:
Updated to `follow-up` to be consistent